src/index.html

```diff
@@ -8,6 +8,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <!-- <h1>Stars</h1> -->
```

Comment:
It is not necessary to comment out the 'h1' tag. Removing the comment would make the code cleaner. If you want to keep the 'h1' element, just uncomment it. 📝